ports/winit/app.rs

```diff
@@ -60,51 +121,107 @@ impl App {
             servo: None,
             suspended: Cell::new(false),
             windows: HashMap::new(),
+            minibrowser: None,
         };
 
+        if opts::get().minibrowser && window.winit_window().is_some() {
+            // Make sure the gl context is made current.
+            let webrender_surfman = window.webrender_surfman();
+            let webrender_gl = match webrender_surfman.connection().gl_api() {
+                GLApi::GL => unsafe { gl::GlFns::load_with(|s| webrender_surfman.get_proc_address(s)) },
+                GLApi::GLES => unsafe {
+                    gl::GlesFns::load_with(|s| webrender_surfman.get_proc_address(s))
+                },
+            };
+            let gl = unsafe {
+                glow::Context::from_loader_function(|s| {
+                    webrender_surfman.get_proc_address(s)
+                })
+            };
+            webrender_surfman.make_gl_context_current().unwrap();
+            debug_assert_eq!(webrender_gl.get_error(), gleam::gl::NO_ERROR,);
```

Comment:
Looks like a trailing comma snuck in after `gleam::gl::NO_ERROR`.

Reply:
fixed in ebdb00783024faa76abea961aced11a746b95ac4

Reply:
fixed in ebdb00783024faa76abea961aced11a746b95ac4